js/controllers/list/ListPanelFilter.vue

```diff
@@ -24,10 +24,34 @@ export default {
 	},
 	methods: {
 		/**
-		 * Emit an event to filter items in the list panel
+		 * Check if a filter is currently active
 		 */
-		filterList: function(data) {
-			this.$emit('filterList', data);
+		isFilterActive: function(type, val) {
+			return this.activeFilters.filter(filter => {
+				return filter.type === type && filter.val === val;
+			}).length
+		},
+
+		/**
+		 * Add a filter
+		 */
+		filterBy: function(type, val) {
+			if (this.isFilterActive(type, val)) {
+				this.clearFilter(type, val);
+				return;
+			}
```

Comment:
It's a toggle feature. Click once to activate it. If you click on the filter again, you deactivate it.